subsurface-mobile-main.cpp

```diff
@@ -54,7 +54,6 @@ int main(int argc, char **argv)
 	prefs.animation_speed = 0; // we render the profile to pixmap, no animations
 
 	// always show the divecomputer reported ceiling in red
-	prefs.dcceiling = 1;
```

Comment:
That comment was wrong - because we always "did not show them". Your patch makes it mostly right (if we show them, show them in red)